src/store/poolSlice.ts

```diff
@@ -368,6 +375,56 @@ export const createPoolSlice: StateCreator<
         user,
       });
     },
+    claimRewards: async ({ selectedReward }) => {
+      // TODO: think about moving timestamp from hook to EventEmitter
+      const timestamp = dayjs().unix();
+      const reserves = selectFormattedReserves(get(), timestamp);
```

Comment:
this is unnecessary, you don't need formatted reserves here, vanilla reserves is just fine.